### ISSUE TITLE
Lots of errors

### ISSUE THREAD

**Issue by Cellolide** [BEEmod/BEE2.4#823]
I was using BEEMod for a long time now and everytime I have installed, opened and extracted stuff to portal the same way, however now I am getting an error after pressing the "Export to Portal 2" button (see BEE2.error.log)
[BEE2.error.log](https://github.com/BEEmod/BEE2.4/files/1670575/BEE2.error.log)

After that the program crashes and if i want to start it again I get a lot of other errors. (see image below).
<img width="362" alt="errors" src="https://user-images.githubusercontent.com/33907318/35476812-c7b6a290-03b6-11e8-96de-df1e733bc99a.png">

In conclusion I can't even open BEEMod without receiving all these errors :(

**Comment by TeamSpen210** [comment]
Check if the `BEE2.ico` file is missing.

**Comment by Cellolide** [comment]
I've seen that the first time on my Owen and completely "reextracted" the BEEMod zip file including packages. So that's definitely not the problem

**Comment by TeamSpen210** [comment]
Ensure there's a copy inside and outside the `bin/` folder.

**Comment by Cellolide** [comment]
Well I can prevent the whole bunch of errors at the startup, but only once. After I press the Export to Portal 2 button the following errors occur again and the BEE.ico as well as the readme.txt get removed from the bee2 folder
<img width="303" alt="unbenannt" src="https://user-images.githubusercontent.com/33907318/35488360-b6ac51f8-0487-11e8-9436-c1abb862fab9.PNG">
